### PR TITLE
Initialized working_zone using new_geometry method instead of from_geojson

### LINKED ISSUE #2101: interventions#compute (JSON::ParserError) "784: unexpected token at 'SRID=4326;MULTIPOLYGO...
Happens when editing an intervention, can be seen in network as soon as the page loads

### PR DESCRIPTION
**Pull request by Zank94**
Fixes #2101

### CODE REVIEW

**Comment by Zank94** [comment]
[ekylibre:debug/gh2101-Fix-interventions-compute-method](https://trello.com/c/QAKAY6cy/307-ekylibredebug-gh2101-fix-interventions-compute-method)

File: lib/pesticide/usage.rb [hunk 1]
```diff
@@ -4,13 +4,14 @@ class Usage
 
     def initialize(attributes = {})
       @name = attributes.delete(:name)
-      @dose = Measure.new(attributes.delete(:dose)) if attributes[:dose]
+      @dose = (attributes[:dose] && attributes[:dose] =~ /\A-?([\,\.]\d+|\d+([\,\.]\d+)?)\s*[^\s]+\z/) ? Measure.new(attributes.delete(:dose)) : attributes.delete(:dose)
```
Comment (burisu) [inline lib/pesticide/usage.rb]: I'm not sure that helps to fix #2101 issue

Comment (Zank94) [inline lib/pesticide/usage.rb]: @burisu Do you want me to create another issue specific to dose measure or to edit the current one to include dose measure alongside working_zone?

**Comment by Zank94** [comment]
Fixes #2136

**Pull request merged**